eventhubs/client/README.md

```diff
@@ -40,26 +40,62 @@ documentation][event_hubs_product_docs]
 </dependency>
 ```
 
-### Obtain a connection string
+### Methods to authorize with Event Hubs
 
 For the Event Hubs client library to interact with an Event Hub, it will need to understand how to connect and authorize
-with it. The easiest means for doing so is to use a connection string, which is created automatically when creating an
+with it.
+
+### Create an Event Hub client using a connection String
+
+The easiest means for doing so is to use a connection string, which is created automatically when creating an
 Event Hubs namespace. If you aren't familiar with shared access policies in Azure, you may wish to follow the
 step-by-step guide to [get an Event Hubs connection string][event_hubs_connection_string].
 
-### Create an Event Hub client
-
-Once the Event Hub and connection string are available, they can be used to create a client for interacting with Azure
-Event Hubs. Create an `EventHubClient` using the `EventHubClientBuilder`:
+Once the connection string is obtained, create an `EventHubClient` using the `EventHubClientBuilder`:
 
 ```java
 String connectionString = "<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>";
-String eventHubName = "<< NAME OF THE EVENT HUB >>";
+String eventHubPath = "<< NAME OF THE EVENT HUB >>";
 EventHubClient client = new EventHubClientBuilder()
-    .connectionString(connectionString, eventHubName)
+    .connectionString(connectionString, eventHubPath)
     .buildAsyncClient();
 ```
 
+### Create an Event Hub client using Microsoft identity platform (formerly Azure Active Directory)
+
+Azure SDK for Java supports an Azure Identity package, making it simple get credentials from Microsoft identity
+platform. First, add the package:
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.0-preview.1</version>
+</dependency>
+```
+
+All the implemented ways to get a credential can be found under the `com.azure.identity.credential` package. The sample below shows how to use an Azure Active Directory (AAD) application client secret to authorize with Azure Event Hubs.
```

Comment:
nit:  "get a credential" sounds a bit awkward to me.  Maybe consider "request a credential" or similar?

Reply:
Thanks! fixed.